### ISSUE TITLE
Feature Request: Video Settings Option

### ISSUE THREAD

**Issue by Firebolt360** [Bestsoft101/Better-Fullscreen#2]
Rather than just the regular fullscreen toggle in video settings, I would love to see either a 3-option toggle between fullscreen, borderless, or no fullscreen. That or maybe a separate borderless toggle. Whatever the case, I feel like it would be useful to have as an option in video settings.

**Comment by Bestsoft101** [comment]
The mod has a config screen where you can enable or disable borderless fullscreen. To open it, you need https://modrinth.com/mod/modmenu
I made sure that the existing buttons in the video settings menu worked, but I don't want to add more since I assume most people use sodium which replaces the screen anyway

**Comment by Firebolt360** [comment]
Sure, yes! That is technically an option to be able to change settings of the mod, but let me rephrase based on my point of view. I'm a modpack developer looking for a new borderless fullscreen mod to give everyday users the option to easily switch between borderless and normal fullscreen seamlessly. I don't expect most users to dig into individual mod settings.

Currently, I use Cubes Without Borders because within Sodium's video settings menu, it provides the 3-option toggle like I mentioned previously. However, I like the customizability of this mod for setting up configs for end users who would use a given modpack. So if this mod had the 3-option toggle that supports the Sodium video settings menu, that would be absolutely perfect. Another bonus option would be to have a toggle in the config menu to turn the 3-option toggle back into the regular vanilla fullscreen toggle. So it can be enabled and disabled. More customization. To demonstrate what I mean though with the 3-option toggle, here's how they personally do it:
<img width="939" alt="Screenshot 2024-10-25 at 5 14 48 PM" src="https://github.com/user-attachments/assets/c0189869-5274-4507-8e58-ee23b95590ee">

**Comment by Bestsoft101** [comment]
Alright

**Comment by Bestsoft101** [comment]
Done
